pom.xml

```diff
@@ -327,6 +327,12 @@ under the License.
 			</dependency>
 
 			<!-- For dependency convergence -->
+			<dependency>
+				<groupId>net.java.dev.jna</groupId>
+				<artifactId>jna</artifactId>
```

Comment:
Where this come from?

Reply:
Dependency convergence, the OpenSearch brings older version

Reply:
So could we exclude it in general? Or do we need it for some tests to work?

What is the bootstrap licensing issue about?